feat(error): spelling_error_constraintset_delete

PreetiKashyap-bmw opened the pull request:
## Description

Steps to reproduce:

Execute the endpoint /api/constraintSet/{constraintSetId} with forceConstraintDelete = False

Observed behavior:

error message: "The resorce is in conflict with the current request."


## Why
Required behavior:

The resource is in conflict with the current request.

## Issue

[DSCP-477](https://atc.bmwgroup.net/jira/browse/DSCP-477)

## Checklist

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/admin/Dev%20Process/How%20to%20contribute.md#commits-branches-and-pull-requests-guidelines)
- [X] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [X] I have created and linked IP issues or requested their creation by a committer
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally
- [X] I have added tests that prove my changes work
- [X] I have checked that new and existing tests pass locally with my changes
- [X] I have commented my code, particularly in hard-to-understand areas

Phil91 commented:
not sure why the workflow wasn't triggered, might be due to the conflicts. but when updating the framework packages please always execute [update_framework_versions](https://github.com/eclipse-tractusx/portal-backend/blob/main/scripts/update_framework_versions.sh) you can find the details in the [update-nuget-packages.md](https://github.com/eclipse-tractusx/portal-backend/blob/main/docs/nuget/update-nuget-packages.md)

please also adjust the description of this issue since one of the open source community don't know about the bmw internal stuff

PreetiKashyap-bmw commented:
closing this PR, added new ticket in portal as this issue pertains to the Tractus-X Portal Backend, and we are consuming the framework package within our internal DSCP environment.  >> https://github.com/eclipse-tractusx/portal-backend/issues/1394  .     
added new PR **https://github.com/eclipse-tractusx/portal-backend/pull/1395** linking with **https://github.com/eclipse-tractusx/portal-backend/issues/1394**
